### ISSUE TITLE
Please help to parser IF function in excel formula

### ISSUE THREAD

**Issue by ericnature** [zaach/jison#384]
For example, IF(A1<>"",A1-1,""). So I want to skip parsing expression 'A1-1' if A1 is empty. How can I do this in jison?

**Comment by Sluchik** [comment]
Hi, @ericnature 

I am trying to find the solution with the "IF" statement too. Do you have any updates?

**Comment by ppqzhao** [comment]
No. Someone suggested that top-down parser can parse such statement but jison is from bottom to top.

**Comment by ericnature** [comment]
No. Someone suggested that parser from top to bottom be good to solve it. But Jison is from bottom to  top parser.

**Comment by Sluchik** [comment]
Thank @ericnature for respond. Do you know some parser from top to bottom which will be suitable for the current case? Currently, I am looking for the best one.

**Comment by ericnature** [comment]
It is said that mid-rule in bison would help. But I did test it. Please refer: https://stackoverflow.com/questions/27160831/how-to-create-a-short-circuit-evaluation-in-bison-using-mid-rules-action
